plugin/src/test/java/io/snyk/plugins/nexus/util/FormatterTest.java

```diff
@@ -46,11 +48,12 @@ void enrichScanResultWithVulnerabilityIssues() {
     ScanResult scanResult = new ScanResult();
 
     // when
-    Formatter.enrichScanResultWithVulnerabilityIssues(scanResult, "3 high, 14 medium, 2 low");
+    Formatter.enrichScanResultWithVulnerabilityIssues(scanResult, "5 critical, 3 high, 14 medium, 2 low");
```

Comment:
since we have old assets already stored, we need to cover both cases: with critical issues and without.